armi/materials/tests/test_materials.py

```diff
@@ -935,16 +935,16 @@ def test_propertyValidTemperature(self):
 
 
 class Lead_TestCase(_Material_Test, unittest.TestCase):
-    """Unit tests for lead materials.
-
-    .. test:: There is a base class for fluid materials.
-        :id: T_ARMI_MAT_FLUID2
-        :tests: R_ARMI_MAT_FLUID
-    """
 
     MAT_CLASS = materials.Lead
 
     def test_volumetricExpansion(self):
+        """Unit tests for lead materials.
+
+        .. test:: There is a base class for fluid materials.
+            :id: T_ARMI_MAT_FLUID2
+            :tests: R_ARMI_MAT_FLUID
+        """
```

Comment:
I think the tag needs to be moved into the `test_linearExpansion()` method, where it is proved that the linear expansion coefficient is zero (which is the defining characteristic of a fluid within the context or ARMI).